.gitlab-ci.yml

```diff
@@ -45,7 +45,7 @@ py36_main:
     - /^never$/
   script:
     - pwd
-    - virtualenv -p python3.6 venv && . venv/bin/activate && make prepare && make develop extras=[all] && pip install htcondor
+    - virtualenv -p python3.6 venv && . venv/bin/activate && pip install -U pip && make prepare && make develop extras=[all] && pip install htcondor
```

Comment:
Maybe `pip install -U pip wheel` ? Most venv's done contain `wheel` and that can make a difference

Reply:
I can add wheel. I don't think not having is is causing any failures though.